one-to-one-sample-app/iOS/SampleApp/MainView.m

```diff
@@ -17,6 +17,8 @@ @interface MainView()
 @property (strong, nonatomic) IBOutlet UIButton *callHolder;
 @property (strong, nonatomic) IBOutlet UIButton *micHolder;
 
+@property (strong, nonatomic) IBOutlet UIButton *reverseCamera;
```

Comment:
Can you change the name to `reverseCameraButton`? Also, do you notice `callHolder` and `micHolder`? Can you change to something like `callButton` and `publisherAudioButton`?